ssg/templates.py

```diff
@@ -27,23 +29,32 @@
     "puppet": templating_lang("puppet", ".pp", template_type.remediation, "puppet"),
     "sce-bash": templating_lang("sce-bash", ".sh", template_type.remediation, "sce")
 }
-preprocessing_file_name = "template.py"
 
-templates = dict()
+PREPROCESSING_FILE_NAME = "template.py"
+TEMPLATE_YAML_FILE_NAME = "template.yml"
 
 
-class Template():
+class Template:
     def __init__(self, template_root_directory, name):
+        self.langs = []
         self.template_root_directory = template_root_directory
```

Comment:
This is a wrong name, it's a root directory of template**s**, not of a template.

Reply:
Yeah.